Purchases/RCPromotionalOffer.h

```diff
@@ -0,0 +1,22 @@
+//
+// Created by César de la Vega  on 2019-04-18.
+// Copyright (c) 2019 Purchases. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import "RCBackend.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface RCPromotionalOffer : NSObject
```

Comment:
Perhaps it would be better to option out this whole class on pre-12.2?

Reply:
I use the class in:

```
- (void)postReceiptData:(NSData *)data
              appUserID:(NSString *)appUserID
              isRestore:(BOOL)isRestore
      productIdentifier:(NSString *)productIdentifier
                  price:(NSDecimalNumber *)price
            paymentMode:(RCPaymentMode)paymentMode
      introductoryPrice:(NSDecimalNumber *)introductoryPrice
           currencyCode:(NSString *)currencyCode
      subscriptionGroup:(NSString *)subscriptionGroup
              discounts:(NSArray<RCPromotionalOffer *> * _Nullable)discounts
             completion:(RCBackendPurchaserInfoResponseHandler)completion
```

so if I opt it out on pre 12.2, RCBackend won't compile